usr.bin/ssh/sshd.c

```diff
@@ -323,6 +320,14 @@ child_reap(struct early_child *child)
 {
 	LogLevel level = SYSLOG_LEVEL_DEBUG1;
 	int was_crash, penalty_type = SRCLIMIT_PENALTY_NONE;
+	const char *child_status;
```

Comment:
set `child_status = ""` here and save two lines below :)

Reply:
i wanted to have the setting in sync, especiially since there is a 2nd case in a for-loop where this would be wrong